lionblog-seongil/src/routes/PostEditPage.jsx

```diff
@@ -1,5 +1,63 @@
+import posts from "../data/posts";
+import { useParams } from "react-router-dom";
 const PostEditPage = () => {
-  return <div>Post Edit Page</div>;
+  const handleSignUpSubmit = () => {
+    alert("수정 완료");
+  };
+  const postId = useParams().postId;
+  const post = posts.find((post) => post.id === parseInt(postId));
+  return (
+    <div className="flex flex-col items-center w-1/2">
+      <h3 className="font-bold text-2xl">게시글 수정</h3>
+      <form className="form gap-2" onSubmit={handleSignUpSubmit}>
+        <label required type="text" className="label">
+          제목:
+        </label>
+        <input
+          required
+          value={post.title}
+          type="text"
+          id="title"
+          className="input"
+        />
+        <label required className="label">
+          내용:
+        </label>
+        <input
+          className="h-64 input"
+          value={post.content}
```

Comment:
여기도 마찬가지로 defaultValue를 쓰면 좋을 것 같아요!
```suggestion
          defaultValue={post.content}
```